UefiCpuPkg/PiSmmCpuDxeSmm/SmmProfile.c

```diff
@@ -711,27 +711,31 @@ InitSmmProfileCallBack (
   IN EFI_HANDLE      Handle
   )
 {
-  EFI_STATUS                 Status;
+  EFI_STATUS                 Status = EFI_SUCCESS;

```

Comment:
EFI_STATUS Status;

Status = EFI_SUCCESS;

Reply:
Sorry, update in next patch.